symbiosis.list

```diff
@@ -3,5 +3,5 @@
 # This is required for the httpd package to build, as it requires
 # symbiosis-common to perform tests.
 #
-deb http://repo.bytemark.co.uk/symbiosis/jessie ./
+deb http://repo.bytemark.co.uk/meta/symbiosis/jessie/latest ./
```

Comment:
This is the NEW SHINY repo where all packages that we require for symbiosis are collated in one place.  For example, we packaged all the ruby gems required for the acme gem (which we also packaged), and they end up in other repos, so we had to collate them.

Previously there was a script in the symbiosis repo that pulled everything from maker2.